src/hal/src/queue/mod.rs

```diff
@@ -70,6 +71,59 @@ pub struct Submission<Ic, Iw, Is> {
     pub signal_semaphores: Is,
 }
 
+/// Information for binding sparse memory using a [command queue][CommandQueue].
+///
+/// The info is sent to the device through the [`bind_sparse`][CommandQueue::bind_sparse] method.
+#[derive(Debug)]
+pub struct BindSparseInfo<Iw, Is, Ib, Io, Ii> {
```

Comment:
note: I filed https://github.com/gfx-rs/gfx/issues/3525 to follow-up on consistency here